tests/Cisco.Otel.Distribution.UnitTests/TracerTests.cs

```diff
@@ -33,4 +34,22 @@ public void HappyPathTest()
         Assert.AreEqual(1, exportedItems.Count);
         Assert.AreEqual("HappyPathSpan", exportedItems.First().DisplayName);
     }
+
+    [Test]
+    public void ResourceSdkVersionTest()
+    {
+        var options = new CiscoOptions("my-cisco-token");
+
+        var tracerProvider = Tracer.Init(options);
+
+        var resource = tracerProvider.GetResource();
+
+        var sdkVersion =
+            resource
+                .Attributes
+                .First(a => a.Key == "telescope.sdk.version")
+                .Value;
+
+        Assert.AreEqual("0.1.0", sdkVersion);
```

Comment:
Maybe just check a valid version or verify it somehow.
because of in the next version (0.2.0 for example) the test will failed

Reply:
This number will only change in the published version on Nuget, so this test will never fail (0.1.0 is the initial version and will never change in the repo code)